### DISCUSSION TITLE
Looking for whole words

### DISCUSSION THREAD

**Discussion by yarons** [WeblateOrg/weblate#13022] [Q&A] [answered]
In the search dialog there's an option to search for the exact term, this will search for strings with the search term only, which means: "target:=term" will only find strings with term as translation, but what if I want to search for a whole word as part of a string?

I thought about using regex and word boundaries but it doesn't work: target:r"בחר\b", I'm not sure how should I achieve this, thanks.

**Comment by StoyanDimitrov** [comment, marked as answer]
What about `target:r"[[:<:]]בחר[[:>:]]"`?

**Reply by yarons** [reply]
It works, it's just weird that the usual word boundary (`\b`) doesn't work, thanks :)

**Reply by Geeyun-JY3** [reply]
Here is the reason why `\b` does not work: https://github.com/WeblateOrg/weblate/discussions/7095#discussioncomment-1954225

**Reply by nijel** [reply]
Let's describe this in the documentation: https://github.com/WeblateOrg/weblate/pull/13085
